lamps-rfc7030-csrattrs.mkd

```diff
@@ -518,8 +569,23 @@ The Base64:
 
 ### ASN.1 DUMP output
 
-Provide a CSR with an EC key from sha521, include your serial number,
-friendly name, and favorite drink, and hash it with SHA512
+The CsrAttrs structure contains:
+
+1. The challengePassword attribute is included to indicate that the
+   CSR should include this value.
+
+2. An ecPublicKey OID is provided with the value secp521r1 to
+   indicate what kind of public key should be submitted.
+
+3. An extensionRequest attribute with a requirement to include the
+   subjectDirectoryAttributes extension.  The friendly name and
+   and favorite drink attributes {{favoritedrink}} are required in the extension.
```

Comment:
This text also needs to be adapted after fixing the encoding of the example as indicated above.